snaps/index.mdx

```diff
@@ -10,6 +10,7 @@ import YoutubeEmbed from '@site/src/components/YoutubeEmbed'
 Snaps is an open source system that allows anyone to safely extend the functionality of
 [MetaMask](https://metamask.io/), creating new web3 end user experiences.
 Get started building your own Snaps by [installing MetaMask Flask](get-started/install-flask.md).
+Further explore by engaging with [tutorials](/learn/tutorials/index.md), [how-to guides](/how-to/index.md), and other [resources](/learn/resources/index.md).
```

Comment:
I'm not sure if this is necessary here, since we provide links to all the features and beginner material later in this page. It might be best to keep this intro paragraph focused, and provide only one call to action (install Flask). If people are interested in other material, they will quickly see the abundance of guides and concepts in the rest of this page.